packages/auth/src/graphql/schema.graphql

```diff
@@ -1,6 +1,9 @@
 type Query {
   "Fetch a page of grants."
-  grants(input: PaginationInput): GrantsConnection!
+  grants(input: PaginationInput, filter: GrantFilter): GrantsConnection!
```

Comment:
@BlairCurrey what do you think about the `input: PaginationInput` object? vs like you have for the webhook events:

https://github.com/interledger/rafiki/blob/b96ddf1a865143a5c87b3705594b3cd5c2f48287/packages/backend/src/graphql/schema.graphql#L57-L68

Reply:
Although my initial instinct was a pagination input like what we have in the `auth` package, we settled on separate `after`, `before`, `first`, `last` args for consistency with the relay spec and conventions around implementing it. I think that's probably still the right approach and ideally we standardize across packages.

Reply:
My opinion isn't very firm here TBH. Other than being consistent. I mostly preferred separate because that's how we saw the relay spec implemented elsewhere and it's also less to break/refactor (but I guess that's not a major consideration). I'm still leaning that way for those reasons but I'm wondering what you think @mkurapov because I think all of our initial instincts were to not use separate args.

Reply:
Even though I agree w @dragosp1011 (I like having the object type instead of the listed arguments), let's stay consistent and have the 4 variables instead. We can then do a swoop of all of the changes in one go to convert to object at some point in the future.